src/c++/perf_analyzer/request_rate_worker.cc

```diff
@@ -89,6 +92,9 @@ RequestRateWorker::CreateContext()
 void
 RequestRateWorker::HandleExecuteOff()
 {
+  // FIXME
```

Comment:
I have removed the FIXME. The issue was that I had to hardcode CTXID to be 0 in many functions. For now I have made a private member variable ctx_id_.  In the future with further refactors, this may go away.